lib/Notification/bar/barTypes.js

```diff
@@ -0,0 +1,16 @@
+import { func, node, string, bool } from 'prop-types';
+
+export const defaults = {
```

Comment:
💚 One other method could be to add these default props and types to `NotificationBarBase`, and then do something like the following in the sub components:
```
// NotificationBarWarning.js
....

NotificationBarWarning.propTypes = NotificationBarBase.propTypes;
```